apps/api/src/modules/report/report.service.spec.ts

```diff
@@ -1,19 +1,38 @@
 import { Test, TestingModule } from '@nestjs/testing'
+import { PrismaService } from 'src/database/prisma.service'
 
+import { ReportRepository } from './report.repository'
 import { ReportService } from './report.service'
 
 describe('ReportService', () => {
   let service: ReportService
+  let repository: ReportRepository
 
   beforeEach(async () => {
     const module: TestingModule = await Test.createTestingModule({
-      providers: [ReportService],
+      providers: [ReportService, ReportRepository, PrismaService],
     }).compile()
 
     service = module.get<ReportService>(ReportService)
+    repository = module.get<ReportRepository>(ReportRepository)
   })
 
   it('should be defined', () => {
     expect(service).toBeDefined()
   })
+
+  describe('postReport', () => {
+    const postReportDTO = {
+      reason: 'Insert reason',
+    }
+
+    it('should post report correctly', async () => {
+      jest.spyOn(repository, 'createReport').mockResolvedValue()
+
+      await service.postReport(1, postReportDTO, 1)
```

Comment:
should we replace `1` with some constant 
ex. MOCK_USER_ID = 1
It look like magic number here and make me confuse when I read it for the first time